ios/google_navigation_flutter/Sources/google_navigation_flutter/MapConfiguration.swift

```diff
@@ -57,6 +58,7 @@ extension MapConfiguration {
       minZoomPreference ?? kGMSMinZoomLevel,
       maxZoom: maxZoomPreference ?? kGMSMaxZoomLevel
     )
+    mapView.padding = padding ?? UIEdgeInsets.zero
```

Comment:
Is default UIEdgeInsets.zero in the mapview?
if not, we should set padding only if padding is not null

Reply:
It probably is but it doesn't seem to be documented. I'll put it behind null check.